### PR TITLE
Fix build issues

### PR DESCRIPTION
**Pull request by petervdonovan**

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -25,21 +25,21 @@ export async function run(softError = false): Promise<string> {
...
 
-    core.info('Building the Lingua Franca compiler...')
-    await build(dir)
-
     configurePath(dir)
 
     core.info('Checking all Lingua Franca files:')
     skipDirs.push(dir)
-    if ((await checkAll('.', ignore)) === false) {
+    if ((await checkAll(dir, '.', ignore)) === false) {
```
Comment (lhstrh) [inline src/main.ts:32]: I don't think this one is right because `dir` refers to the directory of the `lingua-franca` checkout..

Comment (lhstrh) [reply src/main.ts:32]: I think we should parameterize it and have `.` as the default maybe..

Comment (petervdonovan) [reply src/main.ts:32]: It needs to have access to the directory of the `lingua-franca` checkout so that it can run `lfc`

Comment (lhstrh) [reply src/main.ts:32]: But we put `lfc` on the path

Comment (lhstrh) [reply src/main.ts:32]: Let me do a little bit of experimentation. Thank you so much for fixing the build issues!

**Review by lhstrh — APPROVED** [review]
This is great! Thanks for all the help, @petervdonovan!

**Pull request merged**